source/tomopy/prep/normalize.py

```diff
@@ -150,8 +150,8 @@ def normalize(
         flat = np.mean(flat, axis=0, dtype=np.float32)
         dark = np.mean(dark, axis=0, dtype=np.float32)
     elif averaging == 'median':
-        flat = np.median(flat, axis=0, dtype=np.float32)
-        dark = np.median(dark, axis=0, dtype=np.float32)
+        flat = np.median(flat, axis=0)
+        dark = np.median(dark, axis=0)
```

Comment:
```suggestion
        flat = np.median(flat, axis=0).astype(np.float32)
        dark = np.median(dark, axis=0).astype(np.float32)
```

Reply:
Median does not have a dtype parameter, but let's preserve the intention of the code.